README.md

```diff
@@ -4,11 +4,16 @@ This repo is a thin wrapper around the [Pact Broker Gem][pact-broker-gem] that
 allows Pact Broker to be run on unicorn server on the
 [GOV.UK PAAS][government-paas].
 
-It is used by projects such as [Publishing API][publishing-api],
+As PaaS is being decommissioned later this year, we are currently migrating
+Pact Broker to Heroku, and so this repo also contains a `Dockerfile` and
+associated `heroku.yml` file to allow it to run there. After the migration is
+complete, the old wrapper script will be removed from this repo.
```

Comment:
Could you link to the wrapper script?

Reply:
I suppose I should be more explicit, I'm talking about most of the repo here: `config.ru`, `Gemfile`, `Procfile`, essentially everything except `Dockerfile`, `heroku.yml` and the documentation